app/src/main/kotlin/org/fossify/phone/helpers/RecentsHelper.kt

```diff
@@ -61,16 +74,25 @@ class RecentsHelper(private val context: Context) {
             accountIdToSimIDMap[it.handle.id] = it.id
         }
 
+        var selection: String? = null
+        var selectionParams: Array<String>? = null
+
+        val lastId = previousRecents.lastOrNull()?.id
+        if (lastId != null) {
+            selection = "${Calls._ID} < ?"
+            selectionParams = arrayOf(lastId.toString())
+        }
```

Comment:
We should use the `Calls.DATE` instead of `Calls._ID`, it would be more reliable.

Reply:
```kotlin       
 val lastDate = previousRecents.lastOrNull()?.startTS?.times(1000L)
        if (lastDate != null) {
            selection = "${Calls.DATE} < ?"
            selectionParams = arrayOf(lastDate.toString())
        }
```

This should do it.

Reply:
I haven't seen your comment before, I did it a bit different, but it works.